src/data_gradients/dataset_adapters/config/data_config.py

```diff
@@ -117,7 +118,7 @@ def to_json(self) -> JSONDict:
             "is_batch": self.is_batch,
             "image_channels": None if self.image_channels is None else self.image_channels.channels_str,
             "n_classes": self.n_classes,
-            "class_names": self.class_names,
+            "class_id_to_name": self.class_id_to_name,
```

Comment:
Why would we introduce such a big change instead of just extending the current class_names to support mappings ?

Reply:
The change `class_id_to_name` -> `class_names ` is mainly internal (unless someone instantiates the config directly).

The reason why I changed it is because when I first left it as `class_names`, it was quite confusing because this naming convention usually refers to a list.
But yeah, I wasnt sure whether the change of name was a good idea or not, I expected you or Eugene to raise the point.

It's easy to rollback, should we?

Reply:
I think we should, yes.

Reply:
Done here [e941c0d](https://github.com/Deci-AI/data-gradients/pull/205/commits/e941c0db81f34938f83a35ff1b77770ebe75f2a7)